src/App.css

```diff
@@ -17,9 +17,21 @@
   max-width: 500px;
 }
 
+.primary-pane{
+  /* Used to house everything that will sit on the right hand main window.*/
+  flex: 4;
+}
+
 .editor{
   background-color: white;
   color: #282c34;
-  flex: 4;
+  height: 90vh;
 }
 
+.logging{
```

Comment:
You could set this a fixed set of px and then set the editor to flex 1 so it fills out space.